redwood-flexbox-engine/src/commonTest/kotlin/app/cash/redwood/StringWidget.kt

```diff
@@ -21,29 +21,20 @@ package app.cash.redwood
  */
 class StringWidget(
   private val text: String,
-) {
+) : Measurable() {
   private val words = text.split(" ")
 
   private var left = -1
   private var top = -1
   private var right = -1
   private var bottom = -1
 
-  // TODO: fix the Node API to not conflate layout parameters with measurements.
```

Comment:
Fixing this todo is the big motivation here. StringWidget does all the measurement, Node does all the layout parameters.